Html module fails to run task superDev

#### Issue details
Html task `superDev` fails due to gretty not being fetchable from maven.

Fix: https://stackoverflow.com/questions/66304743/getting-error-on-gretty-when-building-html-project-for-libgdx

#### Reproduction steps/code
Create a new project via the setup app and try to run gradle taks `superDev`

#### Version of LibGDX and/or relevant dependencies
1.9.14

#### Stacktrace
```java
 ./gradlew.bat superDev
> Task :html:beforeRun FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':html:beforeRun'.
> Could not resolve all files for configuration ':html:grettyRunnerJetty94'.
   > Could not find org.gretty:gretty-runner-jetty94:3.0.2.
     Searched in the following locations:
       - file:/C:/Users/lyze/.m2/repository/org/gretty/gretty-runner-jetty94/3.0.2/gretty-runner-jetty94-3.0.2.pom
       - https://repo.maven.apache.org/maven2/org/gretty/gretty-runner-jetty94/3.0.2/gretty-runner-jetty94-3.0.2.pom
       - https://dl.google.com/dl/android/maven2/org/gretty/gretty-runner-jetty94/3.0.2/gretty-runner-jetty94-3.0.2.pom
       - https://oss.sonatype.org/content/repositories/snapshots/org/gretty/gretty-runner-jetty94/3.0.2/gretty-runner-jetty94-3.0.2.pom
       - https://oss.sonatype.org/content/repositories/releases/org/gretty/gretty-runner-jetty94/3.0.2/gretty-runner-jetty94-3.0.2.pom
     Required by:
         project :html

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See **https://docs.gradle.org/6.7.1/userguide/command_line_interface.html#sec:command_line_warnings**
```

#### Please select the affected platforms
- [ ] Android
- [ ] iOS
- [x] HTML/GWT
- [ ] Windows
- [ ] Linux
- [ ] MacOS

--- a/build.gradle
+++ b/build.gradle
@@ -25,7 +25,7 @@ allprojects {
             google()
             mavenLocal()
             mavenCentral()
-            maven { url 'https://plugins.gradle.org/m2/' }
+            gradlePluginPortal()
             maven { url "https://oss.sonatype.org/content/repositories/snapshots/" }
         }
         dependencies {
@@ -37,7 +37,7 @@ allprojects {
         google()
         mavenLocal()
         mavenCentral()
-        maven { url 'https://plugins.gradle.org/m2/' }
+        gradlePluginPortal()
         maven { url "https://oss.sonatype.org/content/repositories/snapshots/" }
     }
 }

--- a/build.gradle
+++ b/build.gradle
@@ -18,7 +18,7 @@ apply from: "gradle/dependencies.gradle"
 
 allprojects {
     group = 'com.badlogicgames.gdx'
-    version = '1.9.13-SNAPSHOT'
+    version = '1.9.15-SNAPSHOT'
 
     buildscript {
         repositories {

--- a/extensions/gdx-setup/src/com/badlogic/gdx/setup/BuildScriptHelper.java
+++ b/extensions/gdx-setup/src/com/badlogic/gdx/setup/BuildScriptHelper.java
@@ -75,6 +75,7 @@ public class BuildScriptHelper {
 		write(wr, DependencyBank.mavenLocal);
 		write(wr, DependencyBank.mavenCentral);
 		write(wr, DependencyBank.google);
+		write(wr, "maven { url \"" + DependencyBank.gradlePlugins + "\" }");
 		write(wr, "maven { url \"" + DependencyBank.libGDXSnapshotsUrl + "\" }");
 		write(wr, "maven { url \"" + DependencyBank.libGDXReleaseUrl + "\" }");
 		write(wr, "}");

--- a/extensions/gdx-setup/src/com/badlogic/gdx/setup/BuildScriptHelper.java
+++ b/extensions/gdx-setup/src/com/badlogic/gdx/setup/BuildScriptHelper.java
@@ -34,7 +34,7 @@ public class BuildScriptHelper {
 		write(wr, "repositories {");
 		write(wr, DependencyBank.mavenLocal);
 		write(wr, DependencyBank.mavenCentral);
-		write(wr, "maven { url \"" + DependencyBank.gradlePlugins + "\" }");
+		write(wr, DependencyBank.gradlePlugins);
 		write(wr, "maven { url \"" + DependencyBank.libGDXSnapshotsUrl + "\" }");
 		write(wr, DependencyBank.google);
 		write(wr, "}");
@@ -75,7 +75,7 @@ public class BuildScriptHelper {
 		write(wr, DependencyBank.mavenLocal);
 		write(wr, DependencyBank.mavenCentral);
 		write(wr, DependencyBank.google);
-		write(wr, "maven { url \"" + DependencyBank.gradlePlugins + "\" }");
+		write(wr, DependencyBank.gradlePlugins);
 		write(wr, "maven { url \"" + DependencyBank.libGDXSnapshotsUrl + "\" }");
 		write(wr, "maven { url \"" + DependencyBank.libGDXReleaseUrl + "\" }");
 		write(wr, "}");

--- a/extensions/gdx-setup/src/com/badlogic/gdx/setup/DependencyBank.java
+++ b/extensions/gdx-setup/src/com/badlogic/gdx/setup/DependencyBank.java
@@ -35,7 +35,7 @@ public class DependencyBank {
 	static String mavenLocal = "mavenLocal()";
 	static String mavenCentral = "mavenCentral()";
 	static String google = "google()";
-	static String gradlePlugins = "https://plugins.gradle.org/m2/";
+	static String gradlePlugins = "gradlePluginPortal()";
 	static String libGDXSnapshotsUrl = "https://oss.sonatype.org/content/repositories/snapshots/";
 	static String libGDXReleaseUrl = "https://oss.sonatype.org/content/repositories/releases/";
 

